cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -134,18 +138,52 @@ def migration(self, key, value):
                 "subscription": clean_val("a", value, str),
             }
         )
-    if key == "780__":
-        _relation_previous = clean_val("w", value, str, req=True)
 
-    if key == "785__":
-        _relation_next = clean_val("w", value, str, req=True)
+    _related = _migration.get("related", [])
+    relation_type = None
+    _has_related = _migration.get("has_related", False)
+    # language relation
+    if key == "787__" and "i" in value:
+        relation_language = clean_val("i", value, str)
+        if relation_language:
+            relation_type = LANGUAGE_RELATION.name
+    # has supplement/supplement to relation
+    if key == "770__" or key == "772__" and "i" in value:
+        description = clean_val("i", value, str)
+        if description:
+            relation_type = OTHER_RELATION.name
+    # continues/is continued by relation
+    if key == "780__" or key == "785__" and "i" in value:
+        relation_sequence = clean_val("i", value, str)
+        if relation_sequence:
+            relation_type = SEQUENCE_RELATION.name
+            if key == "780__":
+                sequence_order = "next"
+            else:
+                sequence_order = "previous"
+
+    if relation_type:
+        recid = clean_val("w", value, str, req=True)
+        # split in case of multiple values
```

Comment:
do you have a test for the multiple values use case?

Reply:
yes it's here https://github.com/CERNDocumentServer/cds-ils/blob/246eb6e2a85e9445c65f2be4db1de6e21e3e6f02/tests/migrator/test_journals.py#L414